docs/coordinates/solarsystem.rst

```diff
@@ -0,0 +1,44 @@
+.. include:: references.txt
+
+.. _astropy-coordinates-solarsystem:
+
+Solar System Ephemerides
+----------------------------
+
+`astropy.coordinates` can calculate the |SkyCoord| of some of the major
+solar system objects. This functionality requires the `jplephem` package
```

Comment:
I think you need to either provide the link to the package, since I don't think it's given anywhere otherwise? (and so the above reference will fail).